starter/style.css

```diff
@@ -181,4 +181,9 @@ i {
 .input-wrapper {
     padding: 2.5rem;
     text-align: center;
+}
+
+.input-wrapper h3, 
+.input-wrapper input {
+    margin-bottom: .75rem;
```

Comment:
Why not using a generic class selector instead of relying in wrapped html elements?
Doing so will scale better and will improve performance